Backend/Remora.Discord.Caching/API/CachingDiscordRestUserAPI.cs

```diff
@@ -244,4 +244,64 @@ public async Task<Result<IUser>> GetCurrentUserAsync(CancellationToken ct = defa
 
         return result;
     }
+
+    /// <inheritdoc />
+    public async Task<Result<IApplicationRoleConnection>> GetUserApplicationRoleConnectionAsync
+    (
+        Snowflake applicationID,
+        CancellationToken ct = default
+    )
+    {
+        var key = KeyHelpers.CreateCurrentUserApplicationRoleConnectionCacheKey(applicationID);
+        var cacheResult = await _cacheService.TryGetValueAsync<IApplicationRoleConnection>(key, ct);
+
+        if (cacheResult.IsSuccess)
+        {
+            return cacheResult;
+        }
+
+        var getUserApplicationRoleConnection = await _actual.GetUserApplicationRoleConnectionAsync
+        (
+            applicationID,
+            ct
+        );
+        if (!getUserApplicationRoleConnection.IsDefined(out var userApplicationRoleConnection))
```

Comment:
Spacing

Reply:
It looks a bit odd for sure, but I think it is fine to not add a line break to keep the logical connection between those 2 lines like above.(https://github.com/Remora/Remora.Discord/blob/main/Backend/Remora.Discord.Caching/API/CachingDiscordRestUserAPI.cs#L205-L206)